packages/editor/src/components/collab-sidebar/style.scss

```diff
@@ -0,0 +1,261 @@
+.editor-collab-sidebar {
+	&__activities {
+		padding: 16px;
+		background-color: #f9f9f9;
+	}
+
+	&__thread {
+		position: relative;
+		padding: 16px;
+		border-radius: 8px;
+		border: 1px solid $gray-300;
+		background-color: $gray-100;
+		margin-bottom: 16px;
+
+		.components-base-control__field {
+			margin-bottom: 0;
+		}
+
+		input[type="text"] {
+			@include input-control;
```

Comment:
Please do not use `.components-*` classnames in your styles, as they are not public APIs and are not meant to be used externally to the components.

Also, please do not override "internal" component styles, as it defeats the purpose of using a shared component library.